tasks/rtloader.py

```diff
@@ -16,6 +16,7 @@ def build(ctx, install_prefix=None, cmake_options=''):
     here = os.path.abspath(os.path.dirname(__file__))
     dev_path = os.path.join(here, '..', 'dev')
 
+    print("cmake options {}".format(cmake_options))
```

Comment:
we should remove this since the `-e` option of invoke already print the whole comand